### PR TITLE
Add Uniswap Widget, bumping wagmi to make provider compatible

### PR DESCRIPTION
**Pull request by haydenshively**
This was a pain but I think it's working. Can't use the newest version of the Uniswap Widget because it has some weird redux bug. And even the old version that I pinned is too new to work with our wagmi providers. So I had to go through and migrate wagmi stuff to the latest version.

### CODE REVIEW

File: src/components/pool/SwapTab.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import styled from 'styled-components';
+import tw from 'twin.macro';
+import { useSigner } from 'wagmi';
+import { BlendPoolDrawData } from '../../data/BlendPoolDataResolver';
```
Comment (adamegyed) [inline src/components/pool/SwapTab.tsx]: Unused import

File: src/components/pool/SwapTab.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+import { BlendPoolMarkers } from '../../data/BlendPoolMarkers';
+import { OffChainPoolStats } from '../../data/PoolStats';
+
+import { SwapWidget, Theme } from '@uniswap/widgets'
+import '@uniswap/widgets/fonts.css'
+
+export type DepositTabProps = {
+  poolData: BlendPoolMarkers;
+  offChainPoolStats: OffChainPoolStats | undefined;
```
Comment (adamegyed) [inline src/components/pool/SwapTab.tsx]: Unused prop field, remove it here and from the imports

File: src/components/header/ConnectWalletButton.tsx [hunk 3]
```diff
@@ -11,19 +11,26 @@ import { mapConnectorNameToIcon } from './ConnectorIconMap';
...
-  const { accountData, disconnect } = props;
+  const { address, ensName, activeConnector } = props;
   const [modalOpen, setModalOpen] = useState<boolean>(false);
-  const [{ data: connectData, error: connectError }, connect] = useConnect();
 
-  const ensName: string | undefined = accountData?.ens?.name;
-  const formattedAddr = accountData ? FormatAddress(accountData.address) : '';
-  const buttonText = accountData
+  // const { data: connectData, error: connectError, connect } = useConnect();
+  const { data: connectData, error: connectError, connect, connectors, error, isLoading, pendingConnector } = useConnect({
```
Comment (adamegyed) [inline src/components/header/ConnectWalletButton.tsx]: can remove `connectData`, `error`, `isLoading`, and `pendingConnector`

File: src/components/header/Header.tsx [hunk 4]
```diff
@@ -86,9 +86,12 @@ const NavDropdown = styled.div`
 `;
 
 export default function Header() {
-  const [{ data: accountData }, disconnect] = useAccount({
-    fetchEns: true,
-  });
+  const { address, isConnecting, isDisconnected, connector: activeConnector } = useAccount();
```
Comment (adamegyed) [inline src/components/header/Header.tsx]: Can remove `isConnecting`, `isDisconnected`

File: src/pages/BlendPoolPage.tsx [hunk 5]
```diff
@@ -40,15 +40,6 @@ type PoolParams = {
   pooladdress: string;
 };
 
-export type AccountData = {
-  address: string;
-  connector: Connector<any, any> | undefined;
```
Comment (adamegyed) [inline src/pages/BlendPoolPage.tsx:45]: Since type `Connector` is no longer used, can remove import.

**Review by adamegyed — CHANGES_REQUESTED** [review]
LGTM on functionality and styling. There are a few unused imports that were auto-detected, and I noticed the prop requirements of `SwapTab` needlessly included offchain pool stats. Please fix and this should be good to merge.

**Review by adamegyed — COMMENTED** [review]
Thanks for fixing the linting notices and the prop dependency issue. I noticed something new when re-testing: the styling adjustments for the tab switcher seem to have some odd interaction with the swap modal's tooltip in what looks like mismatched z-indices and possibly some width constraints changing the aspect ratio for token logos.
Screen Shot 2022-09-06 at 2 00 02 PM

Please fix, or add it as a GH issue and move to out-of-scope for this ticket.

**Comment by IanWoodard** [comment]
@haydenshively I can go through and try to fix the styling for you if you would like

**Comment by haydenshively** [comment]
Thanks Ian!

**Pull request merged**